src/Microsoft.DotNet.Build.Tasks/PackageFiles/launchSettings.json

```diff
@@ -0,0 +1,19 @@
+{
+    "profiles": {
+        ".NET Core xUnit Console": {
+            "commandName": "Executable",
+            "executablePath": "$(BinDir)testhost/netcoreapp-$(_bc_OSGroup)-$(ConfigurationGroup)-$(ArchGroup)/dotnet.exe",
```

Comment:
If we can use MSBuild properties here, why can't we just have 1 property for `$(TestExecutablePath)` and set it here? Similar for workingDirectory and DEVPATH. That way we don't have to duplicate the folder structures here. The structures can just be in one place - in MSBuild.

Reply:
right if you look at commit 1 I had it like that but changed it as with that it was configuration dependent, means that if you selected the wrong configuration you couldn't launch the profile. IMO launchSettings should be build configuration independent. The duplication of the folder structure is a small price to pay in comparison to the confusion of users.